src/core/public/chrome/chrome_service.tsx

```diff
@@ -158,6 +161,29 @@ export class ChromeService {
         }
         this.collapsibleNavHeaderRender = render;
       },
+      registerNavLink: (useCase: ChromeUseCase, navLink: ChromeRegistrationNavLink) => {
```

Comment:
registerUserCase? to pair with getUserCase$. Also can add an example here for better understanding

Reply:
Changed to `addNavToGroup`, as for example, we add an example in the README of chrome_service, I suppose that should be good enough.